### ISSUE TITLE
Remove `full_design` lines from functions where it isn't needed

### ISSUE THREAD

**Issue by guilhermejacob** [DjalmaPessoa/convey#230]
When I run the code below
```
year.to.analyze <- 2015
# options( encoding="windows-1252" )


# name the database files in the "MonetDB" folder of the current working directory
pnad.dbfolder <- "~/PNAD/MonetDB"


library(downloader)		# downloads and then runs the source() function on scripts from github
library(survey)			# load survey package (analyzes complex design surveys)
library(MonetDBLite)
library(DBI)			# load the DBI package (implements the R-database coding)


# set R to produce conservative standard errors instead of crashing
# http://r-survey.r-forge.r-project.org/survey/exmample-lonely.html
options( survey.lonely.psu = "adjust" )
# this setting matches the MISSUNIT option in SUDAAN

# load pnad-specific functions (to remove invalid SAS input script fields and postStratify a database-backed survey object)
source_url( "https://raw.githubusercontent.com/ajdamico/asdfree/master/Pesquisa%20Nacional%20por%20Amostra%20de%20Domicilios/pnad.survey.R" , prompt = FALSE )


##############################################
# survey design for a database-backed object #
##############################################

# create survey design object with PNAD design information
# using existing table of PNAD data
sample.pnad <-
  svydesign(
    id = ~v4618 ,
    strata = ~v4617 ,
    data = paste0( "pnad" , year.to.analyze ) ,
    weights = ~pre_wgt ,
    nest = TRUE ,
    dbtype = "MonetDBLite" ,
    dbname = pnad.dbfolder
  )
# note that the above object has been given the unwieldy name of `sample.pnad`
# so that it's not accidentally used in analysis commands.
# this object has not yet been appropriately post-stratified, as necessitated by IBGE
# in order to accurately match the brazilian 2010 census projections

# this block conducts a post-stratification on the un-post-stratified design
# and since the R `survey` package's ?postStratify currently does not work on database-backed survey objects,
# this uses a function custom-built for the PNAD.
y <- 
  pnad.postStratify( 
    design = sample.pnad ,
    strata.col = 'v4609' ,
    oldwgt = 'pre_wgt'
  )

library(convey)
y <- convey_prep( y )

###########################
# variable recode example #
###########################

# force age as numeric
y <- update( y , v8005 = as.integer( v8005 ) )

# create a new income without zeros
y <- update( y , v4621 = ifelse( v4621 <= 0 , 1 , v4621 ) )

# reorder variable light
 y <- update( y , light = 1 )
 y <- update( y , light = ordered( v0219, levels=c( 5, 3, 1), labels = c( "Outra forma" , "Óleo, querosene ou gás de botijão" , "Elétrica (de rede, gerador, solar)" ) ) )

############
# analysis #
############


cos <- list( 170, "Óleo, querosene ou gás de botijão" )
svyafc(~v4621+light, design = y, k = .5, g = 0, cutoffs = cos , na.rm = TRUE )
```

I get:

```
> svyafc(~v4621+light, design = y, k = .5, g = 0, cutoffs = cos , na.rm = TRUE )
Error in .local(conn, statement, ...) : 
  Unable to execute statement 'select v4621, light from pnad2015'.
Server says 'ParseException:SQLparser:SELECT: identifier 'light' unknown'.
Called from: .local(conn, statement, ...)
```

Can you help me with that, @ajdamico ?
 
I can use a workaround, but this should be fixed.

**Comment by guilhermejacob** [comment]
Actually, it isn't necessary. My bad.
It's because I'm running the recodes after `convey_prep` and, since it takes arguments from a not-`updated` version of `full_design`, it will go wrong.
Anyways, just leave it open so that i can drop the unnecessary `full_design` calls in functions like this one.

**Comment by ajdamico** [comment]
thanks.  i should fix that globally -- #231

**Comment by ajdamico** [comment]
hi, i haven't looked at this at all.  if you want a second set of eyes,
please reopen and assign to me?  but no problem if you think it is done :)
 thanx
